src/main/java/core/basesyntax/Colors.java

```diff
@@ -0,0 +1,8 @@
+package core.basesyntax;
+
+public enum Colors {
+    black,
+    white,
+    red,
+    blue
```

Comment:
Enums values names should be in UpperCase
```suggestion
    BLACK,
    WHITE,
    RED,
    BLUE
```